app/forms/hub/create_client_form.rb

```diff
@@ -107,7 +107,9 @@ def self.permitted_params
     def default_intake_attributes
       {
         type: "Intake::GyrIntake",
-        visitor_id: SecureRandom.hex(26)
+        visitor_id: SecureRandom.hex(26),
+        primary_consented_to_service: "yes",
```

Comment:
The IP address field... can/should this be the Hub user IP address at the time the Hub user adds them? I would suppose yes. What do you think?

Daydreaming a little more, we could make this a be a fourth enum option `primary_consented_to_service: "dropoff"`. That's perhaps too much hassle, so ignore this enum comment unless it sparks joy.

The IP address thing I do want your opinion on.

Reply:
Hmm... i dont know. i think it makes sense to leave the IP nil for drop offs, and to use information about the intake and tax returns that show that it is a drop off instead of muddying with another enum option. Especially because then one would need to maintain knowledge that "yes" and "dropoff" are both valid options, and the point of this PR is to decrease overhead of "unique" data states.

Reply:
the `_ip` field implies that the consent was done online, whereas dropoff clients are essentially consenting verbally or in writing / signature to the hub user on-site, depending on the VITA process at that location.